server/gameconnection.py

```diff
@@ -637,26 +637,26 @@ def __str__(self):
 
 
 COMMAND_HANDLERS = {
-    "AIOption":             GameConnection.handle_ai_option,
-    "Bottleneck":           GameConnection.handle_bottleneck,
-    "BottleneckCleared":    GameConnection.handle_bottleneck_cleared,
-    "Chat":                 GameConnection.handle_chat,
-    "ClearSlot":            GameConnection.handle_clear_slot,
-    "Desync":               GameConnection.handle_desync,
-    "Disconnected":         GameConnection.handle_disconnected,
-    "EnforceRating":        GameConnection.handle_enforce_rating,
-    "GameEnded":            GameConnection.handle_game_ended,
-    "GameFull":             GameConnection.handle_game_full,
-    "GameMods":             GameConnection.handle_game_mods,
-    "GameOption":           GameConnection.handle_game_option,
-    "GameResult":           GameConnection.handle_game_result,
-    "GameState":            GameConnection.handle_game_state,
-    "IceMsg":               GameConnection.handle_ice_message,
-    "JsonStats":            GameConnection.handle_json_stats,
-    "LaunchStatus":         GameConnection.handle_launch_status,
-    "OperationComplete":    GameConnection.handle_operation_complete,
-    "PlayerOption":         GameConnection.handle_player_option,
-    "Rehost":               GameConnection.handle_rehost,
-    "TeamkillHappened":     GameConnection.handle_teamkill_happened,
-    "TeamkillReport":       GameConnection.handle_teamkill_report,
+    "AIOption": GameConnection.handle_ai_option,
```

Comment:
I guess this was fixed by `autopep8`, which should be dropped, because `flake8` is being told to ignore this in `setup.cfg`, therefore this change should be reverted